### PR TITLE
button

### PR DESCRIPTION
**Pull request by tompatton2**
Red Button

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <h1>Pages#home</h1>
 <p>Find me in app/views/pages/home.html.erb</p>
+
```
Comment (coryb2424) [inline app/views/pages/home.html.erb]: remove this extra new-line

File: app/assets/stylesheets/components/_button.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  transition: background 0.3s ease;
+}
+.btn-pupp:hover {
+  opacity: 1;
+  background: #BF5B8D;
+  text-decoration: none;
+  color: white;
+}
+
+/* Build your own button classes ! */
```
Comment (coryb2424) [inline app/assets/stylesheets/components/_button.scss]: remove this comment please :)

**Comment by coryb2424** [comment]
Could you add a picture of what the button looks like :)

maybe we could use; http://recordit.co/ to record a gif of the button doing stuff (hover, etc)

**Comment by tompatton2** [comment]
screen shot 2018-08-21 at 5 01 29 pm

**Comment by tompatton2** [comment]
screen shot 2018-08-21 at 5 02 31 pm

**Pull request merged**